crates/editor/src/element.rs

```diff
@@ -1482,6 +1482,46 @@ impl EditorElement {
         axis_pair(horizontal_scrollbar, vertical_scrollbar)
     }
 
+    fn layout_minimap(
+        &self,
+        window: &mut Window,
+        cx: &mut App,
+        snapshot: &EditorSnapshot,
+        bounds: Bounds<Pixels>,
+        scrollbar_width: Pixels,
+    ) -> Option<AnyElement> {
+        match snapshot.mode {
+            EditorMode::Full => {
+                let mut editor = self
+                    .editor
+                    .update(cx, |editor, cx| editor.clone(window, cx));
+
+                editor.mode = EditorMode::Minimap;
+                editor.set_text_style_refinement(TextStyleRefinement {
+                    font_size: Some(px(2.).into()),
```

Comment:
![image](https://github.com/user-attachments/assets/91641947-b70e-48f1-8026-b5947b582e16)

![image](https://github.com/user-attachments/assets/d58ddf91-1f71-465e-9043-f157dc625304)

![image](https://github.com/user-attachments/assets/72c3b9f7-e054-47e4-bfd7-51ee771c0847)


Our underlines are not displayed properly.
Ironically, carets are barely visible.


Seems tolerable to move on with the minimap disabled; maybe we can disable diagnostics for the minimap editor?

Reply:
i've disabled this